utils/dockerutil.py

```diff
@@ -276,6 +276,14 @@ def find_cgroup_from_proc(cls, mountpoints, pid, subsys, docker_root='/'):
                     subsys = _subsys
                     break
 
+        # In Ubuntu Xenial, we've encountered containers with no `cpu`
+        # cgroup in /proc/<pid>/cgroup
+        if subsys == 'cpu' and subsys not in subsystems:
+            for s, m in subsystems.iteritems():
```

Comment:
I could see this coming :blush: first because I think `s` is already reserverd in `pdb`, and also well, because they're single characters. But it had such a limited scope, and I couldn't think of of great alternatives to `subsys` and `subsystem` at the time... Fixing.